tests/functional_tests/ffront_tests/test_interface.py

```diff
@@ -229,37 +227,37 @@ def clashing(inp: Field[..., "float64"]):
         tmp: Field[..., "int64"] = inp
         return tmp
 
-    with pytest.warns(FieldOperatorTypeDeductionError, match=r"type inconsistency"):
-        _ = FieldOperatorParser.apply_to_func(clashing)
+    with pytest.warns(FieldOperatorTypeDeductionError, match="type inconsistency"):
+        _ = FieldOperatorParser.apply_to_function(clashing)
 
 
 def test_call():
-    def identity(x: Field[..., "float64"]):
+    def identity(x: Field[..., "float64"]) -> Field[..., "float64"]:
         return x
 
-    def call(inp: Field[..., "float64"]):
+    def call(inp: Field[..., "float64"]) -> Field[..., "float64"]:
         return identity(inp)
 
-    parsed = FieldOperatorParser.apply_to_func(call)
+    parsed = FieldOperatorParser.apply_to_function(call)
     lowered = FieldOperatorLowering.apply(parsed)
 
     assert lowered.expr == itir.FunCall(
         fun=itir.SymRef(id="identity"), args=[itir.FunCall(fun=DEREF, args=[itir.SymRef(id="inp")])]
     )
 
 
-def test_call_expression():
-    def get_identity():
-        return lambda x: x
+# def test_call_expression():
+#     def get_identity():
+#         return lambda x: x
 
-    def call_expr(inp: Field[..., "float64"]):
-        return get_identity()(inp)
+#     def call_expr(inp: Field[..., "float64"]):
+#         return get_identity()(inp)
 
-    with pytest.raises(
-        FieldOperatorSyntaxError,
-        match=r"functions can only be called directly! \(.*\)",
-    ):
-        _ = FieldOperatorParser.apply_to_func(call_expr)
+#     with pytest.raises(
+#         FieldOperatorSyntaxError,
+#         match=r"functions can only be called directly! \(.*\)",
+#     ):
+#         _ = FieldOperatorParser.apply_to_function(call_expr)
```

Comment:
Is this commented out because we want to recover this test in changed form later?

Reply:
I don't think it is actually required. I will delete it.